src/screens/Customers/CustomerMain.tsx

```diff
@@ -1,228 +1,243 @@
+        toMeter: customers.toMeter.filter((customer: CustomerRecord) => searchedCustomers.has(customer.id)),
+        toCollect: customers.toCollect.filter((customer: CustomerRecord) => searchedCustomers.has(customer.id)),
+        done: customers.done.filter((customer: CustomerRecord) => searchedCustomers.has(customer.id)),
       }
+      setCustomers(newCustomers);
+    } else {
+      setCustomers(defaultCustomers);
     }
   }
 
-  const handleSearchChange = (e: any) => {
-    const searchVal = e.target.value.trim();
-    setSearchValue(searchVal);
+  const exitSearch = () => {
+    setSearchValue("");
+    setCustomers(defaultCustomers);
   }
 
-  /**
-   * onPressHandler for selecting sortBy and filterBy attributes.
-   *
-   * NOTE: For a multi-select, event.target.value returns an array, rather than a string.
-   * The first two lines in this function extract the new selected option, then sets the
-   * state variable accordingly. Source: https://material-ui.com/components/selects/#multiple-select
-   */
-  const handleMenuSelect = (event: React.ChangeEvent<{ value: unknown }>) => {
-    const selectedOptions: string[] = event.target.value as string[];
-    const newlySelectedOption: string = selectedOptions[selectedOptions.length - 1];
-
-    const sortByKeys: string[] = Object.keys(SortBy);
-    if (sortByKeys.includes(newlySelectedOption)) {
-      setSortBy(SortBy[newlySelectedOption as keyof typeof SortBy]);
-      return;
-    }
-
-    const filterByKeys: string[] = Object.keys(FilterBy);
-    if (filterByKeys.includes(newlySelectedOption)) {
-      setFilterBy(FilterBy[newlySelectedOption as keyof typeof FilterBy]);
-      setFilterLabels(labels[newlySelectedOption as keyof typeof FilterBy]);
-      return;
+  const getCustomerStatus = (customer: CustomerRecord) => {
+    if (toMeterCustomersSet.has(customer.id)) {
+      return CustomerStatus.METER;
+    } else if (toCollectCustomersSet.has(customer.id)) {
+      return CustomerStatus.PAYMENT;
     }
+    return undefined;
   }
 
+  const getTabContent = () => {
+      let shownCustomers;
+      if (tabValue === CustomerStatus.METER) {
+        shownCustomers = customers.toMeter;
+      } else if (tabValue === CustomerStatus.PAYMENT) {
+        shownCustomers = customers.toCollect;
+      } else if (tabValue === CustomerStatus.DONE) {
+        shownCustomers = customers.done;
+      } else {
+        shownCustomers = customers.all;
+      }
+
+      return (
+        <div className={classes.tabContent}>
+          {shownCustomers.map((customer: CustomerRecord, index) => (
+              <div key={index}>
+                <CustomerCard
+                  customer={customer}
+                  match={match}
+                  status={getCustomerStatus(customer)}
+                />
+              </div>
+            ))}
+        </div>
+      );
+    };
+
+  const getMeterTabLabel = () => (
+    <div>
+      <FlashOnIcon fontSize='inherit' className={classes.tabIcon} />
```

Comment:
Don't need `fontSize='inherit'` since you're overriding it with your class anyways